main/src/ca/uwaterloo/flix/language/ast/WeededAst.scala

```diff
@@ -119,6 +119,8 @@ object WeededAst {
 
     case class LetRec(ident: Name.Ident, ann: Annotations, mod: Ast.Modifiers, exp1: Expr, exp2: Expr, loc: SourceLocation) extends Expr
 
+    case class LocalDef(ann: Annotations, ident: Name.Ident, fparams: List[FormalParam], declaredTpe: Option[Type], declaredEff: Option[Type], exp1: Expr, exp2: Expr, loc: SourceLocation) extends Expr
```

Comment:
The `ann` field is completely unused through all phases, so maybe we just drop it after `Weeder`? If we ever decide to get a termination check or tail recursion check, it should be somewhat easy to re-add it, since it's handled / checked in the Weeder, just not carried any further.

Reply:
OK. Dump it.